RK-coeff-opt/nonlinear_constraints.m

```diff
@@ -120,3 +120,62 @@
     res = polyval(poly_coef, constrain_emb_stability);
     con = [con, res .* conj(res) - 1];
 end
+
+% Energy Preserving conditions
+% Choose pseudo-energy-preserving q (up to 6)
+q=7;
```

Comment:
```suggestion
q=0;
```
If we add the code here for now without adding the option to set it from the main function in https://github.com/ketch/RK-Opt/blob/master/RK-coeff-opt/rk_opt.m, we should choose a default value that does not influence the main function for now.